doc/user/USAGE.md

```diff
@@ -1,63 +1,130 @@
-# Usage
+# Quickstart Guide
 
-Once F3D has been [installed](INSTALLATION.md), you should be able to open any [supported file](#supported-file-formats),
-by either:
-* Using F3D automatically, from your file manager, by directly opening a file.
-* Running F3D and then dragging and dropping files into it to open them.
-* By running F3D from the terminal with a set of command-line [options](OPTIONS.md).
-* As a [thumbnailer](DESKTOP_INTEGRATION.md) for all supported file formats with certain file managers.
+This guide will help you get started using F3D.
+
+As an overview, you'll learn how to run F3D and open supported files. You'll then learn how to configure basic scene constructions, animations and plugins.
+
+## Prerequisites
+
+To use F3D, you'll need to [install](INSTALLATION.md) F3D. See the [Installation](INSTALLATION.md) page for the latest release package supported by your system.
```

Comment:
double link in the same line seems a bit overkill to me. I'll drop the first one.